modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItSqlOperatorsTest.java

```diff
@@ -102,7 +102,7 @@ public void testArithmetic() {
     public void testAggregates() {
         assertExpression("COUNT(*)").returns(1L).check();
         assertExpression("SUM(val)").returns(1L).check();
-        assertExpression("AVG(val)").returns(1).check();
+        assertExpression("AVG(val)").returns(new BigDecimal("1.0000000000000000")).check();
```

Comment:
we can use `setScale` method on DECIMAL value, like this
``BigDecimal.ONE.setScale(16, RoundingMode.UNNECESSARY)``.
For me, it looks more readable.
WDYT?

Reply:
replaced big literals with `setScale`